CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 1.70.0
+* Fix: flaky TestMetricsDisabled unit test. #3588
+* Enhancement: TestGetHostPortRange unit test deterministic. #3587
+* Enhancement: update go module dependencies. #3593
+* Enhancement: Resolve CI NodeJS 12 deprecation warnings for actions/checkout. #3595
+* Enhancement: update docker client library to latest. #3598
+* Enhancement: Provide imageDigest for images from all container repositories. #3576
+* Enhancement: explicitly provide the network name override to nat when using bridge network mode. #3564
+* Enhancement: Support dynamic host port range assignment for singular ports. The dynamic host port range can be configured with ECS_DYNAMIC_HOST_PORT_RANGE in ecs.config; if there is no user-specified ECS_DYNAMIC_HOST_PORT_RANGE, ECS Agent will assign host ports within the default port range, based on platform and Docker API version, for containers that do not have user-specified host ports in task definitions. #3601
+* Dependency Changes. #3602
```

Comment:
I think we can just have a single changelog line for 4 of these PRs (all of 3602, 3593, 3595, 3598 are just dependfency bumps)